From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Zeno Albisser <zeno.albisser@digia.com>
Date: Thu, 1 Aug 2013 14:03:14 +0200
Subject: remove -Wno-deprecated-register from common.gypi.

This warning is not supported by recent clang versions
shipping with XCode. Therefore replacing it with -Wno-deprecated.
This is a bit a too broad fix, but the best we can do for now.
It seems that the flag is still supported in upstream clang.
---
 build/common.gypi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/common.gypi b/build/common.gypi
index ab759282..cf5b196 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -3222,7 +3222,7 @@
               # Clang considers the `register` keyword as deprecated, but e.g.
               # code generated by flex (used in angle) contains that keyword.
               # http://crbug.com/255186
-              '-Wno-deprecated-register',
+              '-Wno-deprecated',
             ],
             'cflags!': [
               # Clang doesn't seem to know know this flag.
@@ -3925,7 +3925,7 @@
                 # Clang considers the `register` keyword as deprecated, but e.g.
                 # code generated by flex (used in angle) contains that keyword.
                 # http://crbug.com/255186
-                '-Wno-deprecated-register',
+                '-Wno-deprecated',
               ],
             }],
             ['use_libcpp==1', {
-- 
1.8.5.3

